From 603e5e6f99a4668caab4f9dea5828ad3e632675a Mon Sep 17 00:00:00 2001
From: Timple Raj M <timple@qti.qualcomm.com>
Date: Tue, 13 Jan 2026 18:44:10 +0530
Subject: [PATCH] arm: dts: ipq5210: Add RDP build support

This change adds the build support for the following RDP's:
  RDP498, RDP499, RDP500, RDP501, RDP502,
  RDP503, RDP504, DB-HM01.1 & DB-HM02.1

Change-Id: I8e5a5d66e15acb697a66513f965559e25437cadd
Signed-off-by: Timple Raj M <timple@qti.qualcomm.com>
---
 arch/arm/dts/Makefile | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 78c13266875..0ebe954fa16 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -1,6 +1,16 @@
 # SPDX-License-Identifier: GPL-2.0+
 
-dtb-$(CONFIG_TARGET_IPQ5210) += ipq5210-emulation.dtb
+dtb-$(CONFIG_TARGET_IPQ5210) += ipq5210-emulation.dtb \
+				ipq5210-rdp497.dtb \
+				ipq5210-rdp498.dtb \
+				ipq5210-rdp499.dtb \
+				ipq5210-rdp500.dtb \
+				ipq5210-rdp501.dtb \
+				ipq5210-rdp502.dtb \
+				ipq5210-rdp503.dtb \
+				ipq5210-rdp504.dtb \
+				ipq5210-db-hm01.1.dtb \
+				ipq5210-db-hm02.1.dtb
 
 dtb-$(CONFIG_TARGET_SMARTWEB) += at91sam9260-smartweb.dtb
 dtb-$(CONFIG_TARGET_TAURUS) += at91sam9g20-taurus.dtb
-- 
2.34.1

